tests/ops/integration_tests/test_mongo_task.py

```diff
@@ -847,7 +847,9 @@ async def test_array_querying_mongo(
     )
     # Returns fields_affected for all possible targeted fields, even though this identity only had some
     # of them actually populated
-    assert sorted(customer_detail_logs[0].fields_affected, key=lambda e: e["field_name"]) == [
+    assert sorted(
```

Comment:
Hah, oh no, did I not run `black`? This was my change 😢

Reply:
it's fine I got it!